etc/ha_healthchecker/ha_healthchecker.py.j2

```diff
@@ -25,9 +28,15 @@ CONFIG_LOCAL = {
 {% if use_test_account %}
     'github_repo': "{{ test_repo_name }}",
     'github_user_token': "{{ test_github_token }}",
+    'github_user_passwd': "{{ test_github_password }}",
```

Comment:
So here will be better to use github auth token instead?

Reply:
There is no way to use oauth token to update the github app webhook, github api doesnt provider that. So we use python crawlers to simulate the normal human to login Github and change the webhook on a browser..